resource-management/pkg/aggregrator/aggregator.go

```diff
@@ -63,6 +63,7 @@ type PullDataFromRRM struct {
 const (
 	DefaultBatchLength = 20000
 	httpPrefix         = "http://"
+	defaultPullInterval = time.Duration(10 * time.Millisecond) // 10ms as default pull interval
```

Comment:
time.Duration() is an unnecessary conversion

Reply:
change is done.